### PR TITLE
1st commit of OSPool Containers

### PR DESCRIPTION
**Pull request by osg-cat**

### CODE REVIEW

**Comment by osg-cat** [comment]
@matyasselmeci, I incorporated some of your comments as I could make sense of them. If you have further comments, changes, etc., I might suggest getting this version in, more or less intact, than doing subsequent PRs. Then there will be a nice history.

File: docs/software/ospool-containers.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+Most, but certainly not all, OSPool payload jobs that request GPUs also request to run within a container.
+This is probably due to GPU-using software stacks often being complex,
+and so users often turn to containers for their runtime environment,
+including ones that we and others pre-build.
+
+Specifically, the Backfill Container images do not include the NVIDIA CUDA drivers
+(although, we are experimenting with that),
+so a site that contributes capacity using Backfill Containers
+and that wants to run GPU jobs _must_ run them under Apptainer.
```
Comment (matyasselmeci) [inline docs/software/ospool-containers.md:91]: I was somewhat wrong about this (per the discussion in Slack earlier today). We do not include the CUDA drivers but instead we mount them in from outside (so we use the CUDA drivers the admin installed on the system outside the pilot).  This works everywhere but Expanse (a phrase you will hear often); the experimental images are for getting GPUs working for Expanse.

Hence it is _not_ true that GPU jobs _must_ be run under Apptainer, though your previous paragraph still applies...

Comment (osg-cat) [reply docs/software/ospool-containers.md]: Ok, I made some changes to try to address this point.

**Review by matyasselmeci — APPROVED** [review]
LGTM!

**Pull request merged**